modules/apps/frontend-js/frontend-js-svg4everybody-web/src/main/java/com/liferay/frontend/js/svg4everybody/web/internal/servlet/taglib/SVG4EverybodyTopHeadDynamicInclude.java

```diff
@@ -78,7 +88,14 @@ public void include(
 			}
 		}
 
-		if (cdnHostEnabled) {
+		ThemeDisplay themeDisplay =
+			(ThemeDisplay)httpServletRequest.getAttribute(
+				WebKeys.THEME_DISPLAY);
+
+		ThemeSpritemapCET themeSpritemapCET = _getThemeSpritemapCET(
+			themeDisplay.getLayout());
```

Comment:
Don't we have [this already](https://github.com/liferay/liferay-portal/blob/8e3c4923ba7338672ea03a796a26d88eaca8580a/modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/internal/events/ClientExtensionsServicePreAction.java#L102-L106)?

Reply:
We do have that in the PreAction, but we need to also grab it here to check the configuration of the CET of svg4everybody. In the PreAction, we are grabbing the CET to set the path on the themeDisplay. So it is slightly duplicated logic, but we do need to grab the configuration in both places